pq-crypto/bike_r3/Makefile

```diff
@@ -0,0 +1,23 @@
+
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+SRCS=$(wildcard *.c)
```

Comment:
Hmmm, did some digging and it looks like [those flags were added as a workaround for a specific gcc bug](https://github.com/aws/s2n-tls/blame/main/pq-crypto/bike_r1/Makefile#L11), and were [first added in a PR that enabled SAW proofs for BIKE Round 1.](https://github.com/aws/s2n-tls/pull/1288)

I don't think we need it right now, but if we ever want to add SAW proofs in the future we might. I'll go ahead and add it now so that we don't stumble up on it later.